pkg/ingester/flush.go

```diff
@@ -248,15 +248,18 @@ func (i *Ingester) flushUserSeries(userID string, fp model.Fingerprint, immediat
 }
 
 func (i *Ingester) collectChunksToFlush(instance *instance, fp model.Fingerprint, immediate bool) ([]*chunkDesc, labels.Labels, *sync.RWMutex) {
-	stream, ok := instance.streams.LoadByFP(fp)
+	var stream *stream
+	var ok bool
+	stream, ok = instance.streams.LoadByFP(fp)
```

Comment:
It's just stylistic I believe.

Reply:
Actually I added a `WithRLock` here at first, then I realized that this `LoadByFP` doesn't need a read-lock here. Because this goroutine is flushing right now, and it doesn't mind more chunks be added before locking. So I removed `WithRLock` later, but forgot to change the declaration here.